lib/shoryuken/manager.rb

```diff
@@ -77,7 +77,11 @@ def processor_done(queue, processor)
 
     def processor_died(processor, reason)
       watchdog("Manager#processor_died died") do
-        logger.error { "Process died, reason: #{reason}" }
+        if reason
+          logger.error { "Process died, reason: #{reason}" }
+        else
+          logger.info { "Process stopped by user" }
```

Comment:
hm @mariokostelac the `trap_exit/reason` isn't well documented. I was wondering if we could use something more generic "Process died", just to avoid saying "by user" in case it wasn't because of a user sending signals, that can be also misleading. WDYT?